examples/pytorch/nlp/huggingface_models/text-to-image/inference/README.md

```diff
@@ -0,0 +1,36 @@
+Step-by-Step
+============
+This document describes the step-by-step instructions to benchmark stable diffusion on Intel® Xeon® with PyTorch and Intel® Extension for PyTorch.
+
+The script ```run_tti.py``` is based on [runwayml/stable-diffusion-v1-5](https://huggingface.co/runwayml/stable-diffusion-v1-5) and provides inference benchmarking.
+
+# Prerequisite
+## Create Environment
+```
+conda install mkl mkl-include -y
+conda install jemalloc gperftools -c conda-forge -y
+pip install torch==1.13.1 --extra-index-url https://download.pytorch.org/whl/cpu
+pip install intel_extension_for_pytorch=1.13.0
+pip install -r requirements.txt
+```
+## Setup variable to speedup
+```
+export KMP_BLOCKTIME=1
+export KMP_SETTINGS=1
+export KMP_AFFINITY=granularity=fine,compact,1,0
+
+# IOMP
+export OMP_NUM_THREADS=< Cores to use >
+export LD_PRELOAD=${LD_PRELOAD}:${CONDA_PREFIX}/lib/libiomp5.so
+# Jemalloc
+export LD_PRELOAD=${LD_PRELOAD}:${CONDA_PREFIX}/lib/libjemalloc.so
+export MALLOC_CONF="oversize_threshold:1,background_thread:true,metadata_thp:auto,dirty_decay_ms:9000000000,muzzy_decay_ms:9000000000"
+```
+
+# Performance
+```bash
+numactl -m <node N> -C <cpu list> python run_tti.py
```

Comment:
change to run_sd_bf16.py? do you want to expose precision as a parameter？ We want to add FP32 precision inference also?

Reply:
or you can create a separate file run_sd_fp32.py